go.mod

```diff
@@ -4,6 +4,7 @@ go 1.16
 
 require (
 	github.com/aws/aws-sdk-go v1.42.4
+	github.com/perlogix/libdetectcloud v0.0.0-20210504034315-50bebf73878e // indirect
```

Comment:
Can you make sure that we lock the version of this external dependency to the one we've reviewed together right now?

Reply:
removed this external dep